common/config/rush/common-versions.json

```diff
@@ -83,23 +83,29 @@
       "~0.6.1" // API Extractor is using an older version of source-map because newer versions are async
     ],
 
-    "webpack": ["~5.35.1"],
+    "webpack": ["~4.44.2"],
 
     // Use two different versions of @types/webpack-dev-server to allow pnpmfile.js to bring in
     // two different versions of the webpack typings
     "@types/webpack-dev-server": ["3.11.3"],
```

Comment:
Is this still necessary? `pnpmfile.js` no longer does anything special with `@types/webpack-dev-server`, and the latest versions of webpack handle their own types. I think this can (and should) be removed as part of the change to `pnpmfile.js`.

Reply:
`common-versions.json` is used to generate a layer in `.pnpmfile.cjs` that will use the preferred version any time it satisfies the semver range specified for a dependency anywhere in the tree, but it makes an exception for entries in `allowedAlternativeVersions`.

Reply:
Looks like we can safely remove `@types/webpack-dev-server` 👍